org-formation/070-guard-duty/GuardDuty.yaml

```diff
@@ -0,0 +1,107 @@
+# From https://github.com/org-formation/org-formation-reference/tree/master/src/templates/070-guard-duty
+AWSTemplateFormatVersion: '2010-09-09-OC'
+
+Parameters:
+  resourcePrefix:
+    Type: String
+
+  accountId:
+    Type: String
+    Description: The identifier from the account used to manage GuardDuty
+
+Resources:
+
+  GuardDutyBucket:
+    Type: AWS::S3::Bucket
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E1012
+    OrganizationBinding: !Ref LogArchiveBinding
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName: !Sub '${resourcePrefix}-guardduty-finding'
+      AccessControl: Private
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+
+  GuardDutyBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E1012
+    OrganizationBinding: !Ref LogArchiveBinding
+    Properties:
+      Bucket: !Ref GuardDutyBucket
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: GuardDutyWrite
+            Effect: Allow
+            Principal:
+              Service: guardduty.amazonaws.com
+            Action:
+              - s3:PutObject
+              - s3:GetBucketLocation
+            Resource:
+              - !Sub 'arn:aws:s3:::${GuardDutyBucket}/*'
+              - !GetAtt GuardDutyBucket.Arn
+
+  Detector:
+    OrganizationBinding: !Ref AllBinding
+    Type: AWS::GuardDuty::Detector
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E1012
+    Properties:
+      Enable: true
+      FindingPublishingFrequency: FIFTEEN_MINUTES
+      Features:
+        - Name: EBS_MALWARE_PROTECTION
+          Status: ENABLED
```

Comment:
this file was copied from `https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/GuardDuty/guard-duty.yaml` and this was the only addition to it.